examples/meta/src/evaluation/cross_validation.sg

```diff
@@ -0,0 +1,94 @@
+#  Cross Validation for Binary Classifier
+CSVFile f_feats_train("../../data/classifier_binary_2d_linear_features_train.dat")
+CSVFile f_feats_test("../../data/classifier_binary_2d_linear_features_test.dat")
+CSVFile f_labels_train("../../data/classifier_binary_2d_linear_labels_train.dat")
+CSVFile f_labels_test("../../data/classifier_binary_2d_linear_labels_test.dat")
+Math:init_random(2)
+
+#![create_features]
+Features features_train = features(f_feats_train)
+BinaryLabels labels_train(f_labels_train)
+Features features_test = features(f_feats_test)
+BinaryLabels labels_test(f_labels_test)
+#![create_features]
+
+#OK
+#![set_parameters]
+real C = 1.0
+real epsilon = 0.001
+#![set_parameters]
+
+#OK
+#![create_instance]
+Machine svm = machine("LibLinear", labels=labels_train)
+svm.put("liblinear_solver_type", enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC)
+svm.put("epsilon", epsilon)
+#![create_instance]
+
+#![create_cross_validation]
+StratifiedCrossValidationSplitting splitting_strategy(labels_train, 2)
+AccuracyMeasure evaluation_criterion()
+CrossValidation cross(svm, features_train, labels_train, splitting_strategy, evaluation_criterion)
+cross.set_autolock(False)
+cross.set_num_runs(2)
+#![create_cross_validation]
+
+#![evaluate_and_get_result]
+CrossValidationResult result = CrossValidationResult:obtain_from_generic(cross.evaluate())
+#![evaluate_and_get_result]
+
+#![get_results]
+real mean = result.get_real("mean")
+real stddev = result.get_real("std_dev")
+#![get_results]
+
+#![get_results_test_data]
+svm.train(features_train)
+BinaryLabels labels_predict = svm.apply_binary(features_test)
+real accuracy_test = evaluation_criterion.evaluate(labels_predict, labels_test)
+#![get_results_test_data]
+
+# Cross Validation for Linear Regression
+CSVFile reg_feats_train("../../data/regression_1d_linear_features_train.dat")
+CSVFile reg_feats_test("../../data/regression_1d_linear_features_test.dat")
+CSVFile reg_lab_train("../../data/regression_1d_linear_labels_train.dat")
+CSVFile reg_lab_test("../../data/regression_1d_linear_labels_test.dat")
```

Comment:
for this example, same comments as for the above

Reply:
Done in next commit.